### DISCUSSION TITLE
Does pugixml has arm64 version for windows OS?

### DISCUSSION THREAD

**Discussion by keiutou** [zeux/pugixml#411] [General]
I found there are Win32 and x64 versions. 
Does pugixml has arm64 version for windows OS?

Thanks,
Yutao

**Comment by zeux** [comment]
No but you can easily build it yourself, it's just a single source file. You should also be able to add an ARM configuration to existing Visual Studio projects without issues.

**Reply by keiutou** [reply]
Thanks for your quick reply. yes, I can create that in my dev box.

Thanks!
